index.html

```diff
@@ -7468,14 +7468,14 @@ <h3>Introduction</h3>
       <h2>Archive sites</h2>
 
       <p>This specification can be found at <a href=
-      "http://www.w3.org/TR/2003/REC-PNG-20031110/index.html"><code>http://www.w3.org/TR/2003/REC-PNG-20031110/index.html</code></a>.</p>
+      "https://www.w3.org/TR/2003/REC-PNG-20031110/index.html"><code>https://www.w3.org/TR/2003/REC-PNG-20031110/index.html</code></a>.</p>
```

Comment:
That is old text, and now incorrect (points to second edition). It adds no value because the top of the spec also has the url, so we can just delete it.

Reply:
OK, but I note that removing this means we'll have an empty section E.1